mdata/store_cassandra.go

```diff
@@ -263,7 +265,9 @@ func (c *CassandraStore) Add(cwr *ChunkWriteRequest) {
 		sum += int(char)
 	}
 	which := sum % len(c.writeQueues)
+	c.writeQueueMeters[which].Value(len(c.writeQueues[which]))
 	c.writeQueues[which] <- cwr
+	c.writeQueueMeters[which].Value(len(c.writeQueues[which]))
```

Comment:
this is not needed.  If processWriteQueue() is not blocked, then it will update this meter every time it reads from the channel.  If it is blocked, then the length of the queue will always just be 1 more then the length before we added to it.